database.py

```diff
@@ -5,33 +5,43 @@ def __init__(self, app):
         self.mongo = PyMongo(app)
 
     def set_username(self, id, username):
-        self.mongo.db.user.insert_One({"username": username})
+        self.mongo.db.user.update_one({"id": id}, {"$set": {
+            "username": username
+        }
+        })
 
     def get_username(self, id, username):
-        return self.mongo.db.user.findOne({"username": username})
+        return self.mongo.db.user.find({"id": id, "username": username})
 
     def set_password(self, id, password):
-        self.mongo.db.user.insert_One({"password": password})
+        self.mongo.db.user.update_one({"id": id}, {"$set": {
+            "password": password
+        }
+        })
 
     def get_password(self, id, password):
-        return self.mongo.db.user.findOne({"password": password})
+        return self.mongo.db.user.find({"id": id, "password": password})
```

Comment:
Should be `find_one`: https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html?highlight=update#pymongo.collection.Collection.find_one